### PR TITLE
Blog and site updates for Beam 2.57.0 release

### PR DESCRIPTION
**Pull request by kennknowles**
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [x] Update `CHANGES.md` with noteworthy changes.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Review by liferoad — APPROVED** [review]

File: website/www/site/content/en/blog/beam-2.57.0.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    should be updated to use new snake_case parameter names.
+* Upgraded Jackson Databind to 2.15.4 (Java) ([#26743](https://github.com/apache/beam/issues/26743)).
+  jackson-2.15 has known breaking changes. An important one is it imposed a buffer limit for parser.
+  If your custom PTransform/DoFn are affected, refer to [#31580](https://github.com/apache/beam/pull/31580) for mitigation.
+
+For the most up to date list of known issues, see https://github.com/apache/beam/blob/master/CHANGES.md
+
+## List of Contributors
+
+According to git shortlog, the following people contributed to the {$RELEASE_VERSION} release. Thank you to all contributors!
```
Comment (tvalentyn) [inline website/www/site/content/en/blog/beam-2.57.0.md:83]: I suspect this has not been updated based on the fact that template is present and absence of recent merged PRs for some people mentioned below.

Comment (liferoad) [reply website/www/site/content/en/blog/beam-2.57.0.md:83]: I thought this list was created by running some scripts.

Comment (kennknowles) [reply website/www/site/content/en/blog/beam-2.57.0.md:83]: I did generate the list of people, with `git log --pretty="%aN" v2.57.0-RC1 ^v2.56.0 | sort -u` and then some manual fixups of obvious duplicates.

I did miss that template variable though.

Comment (kennknowles) [reply website/www/site/content/en/blog/beam-2.57.0.md]: Updated two occurrences of `${RELEASE}` and confirmed there is no more occurence of `$` in the file.

**Pull request merged**